tests/testthat/test-slice.R

```diff
@@ -431,8 +437,10 @@ test_that("slice has informative error messages", {
     vec_slice(1:2, 3L)
     vec_slice(1:2, -3L)
 
-    "# Can't be matrix"
+    "# vec_slice throws error with non-vector subscripts"
     vec_slice(1:3, matrix(TRUE, ncol = 1))
```

Comment:
You're either missing checks here, or you should pull them out of the `verify_errors()` block. The goal is to keep verify-errors blocks in sync with the corresponding block in the output check.